app/views/layouts/LanguageChecker.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views.layouts
+
+import featureswitch.core.config.{FeatureSwitching, WelshLanguage}
+
+class LanguageChecker extends FeatureSwitching {
+
+  def isWelshAvailable: Boolean = isEnabled(WelshLanguage)
```

Comment:
Don't think there is much of a need for a separate file to do this. You could update appConfig to extend FeatureSwitching and then anything pulling it in will have access to isEnabled, makes it easier to deal with newly added features too.